app/navigation/TransportArrangerNavigator.scala

```diff
@@ -26,6 +27,7 @@ import play.api.mvc.Call
 
 import javax.inject.Inject
 
+//noinspection ScalaStyle
```

Comment:
why?

Reply:
Cyclomatic complexity was more than 10, in other navigators this is present. I could split the routing logic into a private method but wanted to keep it consistent with other navs. Up to you?

Reply:
Nah that's fine, just wanted to double check it wasn't a temporary thing that slipped in 🙂